lib/src/commit_builder.rs

```diff
@@ -27,7 +27,9 @@ pub struct CommitBuilder<'repo> {
     rewrite_source: Option<Commit>,
 }
 
-pub fn new_change_id() -> ChangeId {
+/// TODO: This function is still used in some tests, but shouldn't be. Its
+/// output is not reproducible. Its replacement is available in a follow-up commit.
+pub fn new_change_id_deprecated() -> ChangeId {
     ChangeId::from_bytes(Uuid::new_v4().as_bytes())
 }
```

Comment:
This is so short that I'd rather just put a copy of it in the `tests` module in `index.rs`. Then you can just delete this copy in the next commit.